core/domain/uniquekey.go

```diff
@@ -77,10 +77,11 @@ func MustUniqueKey(sbt smartblock.SmartBlockType, key string) UniqueKey {
 	return uk
 }
 
+// and here
 func UnmarshalUniqueKey(raw string) (UniqueKey, error) {
 	parts := strings.Split(raw, uniqueKeySeparator)
 	if raw == "" || len(parts) > 2 {
-		return nil, errors.New("invalid key format")
+		return nil, errors.New(fmt.Sprintf("invalid key format: %s", raw))
```

Comment:
why do you need to do this? If you need to debug what `raw` is, then it's already available in a context of the function call.

Btw, there is a function `fmt.Errorf` for error formatting